assets/kube-scheduler/config.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubescheduler.config.k8s.io/v1beta1
```

Comment:
v1beta1 should be correct for 4.6 (and 4.7)

Reply:
@csrwng During my testing, when using `v1beta1` here, I would get the following error in the openshift-scheduler:
```
no kind "KubeSchedulerConfiguration" is registered for version "kubescheduler.config.k8s.io/v1beta1" in scheme "k8s.io/kubernetes/pkg/scheduler/apis/config/scheme/scheme.go:31"
```